pkg/broker/receiver.go

```diff
@@ -40,6 +42,22 @@ const (
 	writeTimeout = 1 * time.Minute
 )
 
+var (
+	// These MUST be lowercase strings, as they will be compared against lowercase strings.
+	forwardHeaders = sets.NewString(
+		// tracing
+		"x-request-id",
+	)
+	// These MUST be lowercase strings, as they will be compared against lowercase strings.
+	forwardPrefixes = []string{
```

Comment:
use sets or an array, not both? why both?

Reply:
I use a set for the first one so that I can easily do `forwardHeaders.Has(n)`. The second, I iterate over, so slice and set are the same to me.